src/WebDriver/HasElementTrait.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Silverorange\Turing\WebDriver;
+
+/**
+ * @package   Turing
+ * @copyright 2017-2018 silverorange
+ * @license   http://www.gnu.org/copyleft/lesser.html LGPL License 2.1
+ */
+trait HasElementTrait
```

Comment:
Is it just me or is it weird that this trait appears to wholly depend on another trait. Any reason we just don't combine them?

Reply:
This is so we can use the trait in both `Component` and the base `AbstractTest`.

Reply:
Got it.